src/modules/fundingManager/bondingCurve/abstracts/RedeemingBondingCurveBase_v1.sol

```diff
@@ -239,16 +239,6 @@ abstract contract RedeemingBondingCurveBase_v1 is
         // Cache Collateral Token
         IERC20 collateralToken = __Module_orchestrator.fundingManager().token();
 
-        // Require that enough collateral token is held to be redeemable
-        if (
-            (collateralRedeemAmount + projectCollateralFeeCollected)
```

Comment:
How would this be handled in case a project is not collecting the project fee directly but storing it in the contract. I.e. the redeem amount might be taken from the project collateral fee stored in the contract

Reply:
I would say that lies in the responsibility of the functions that are able to transfer collateral tokens out of the contract. In this case the _handleCollateralTokensAfterSell and transferOrchestratorToken in case its a fundingmanager.
The problem is that I dont see a clear way to enforce this properly.

Reply:
I had a talk with @marvinkruse about this.
We agreed to enforce the rule of "there has to be enough funds in the BondingCurve to enable the withdrawal of Project fees". This makes the check viable again (without the redeem amount of course).
In the future we probably still have to talk about the Fee System as a whole, as this was a bigger discussion point overall

Reply:
Makes sense. However the way it is implemented atm isn't right I think. Let's do a quick sync around this tomorrow @FHieser.